site-content/process-native-protocol-specs-in-docker.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# Exit immediately if a command exits with a non-zero status.
+set -e
+
+# Variables
+GO_VERSION="1.20.6"
```

Comment:
Go 1.20.6 is no longer supported. Please bump to the latest version — 1.23.1.

Going forward, it would be good to set it up in a way so that dependabot or renovatebot could open pull requests to upgrade the version (I don't know if one of those tools is used in this repository though)

Reply:
Thank you, fixed.